umh.docs.umh.app/content/en/docs/features/data-connectivity-node-red/index.md

```diff
@@ -18,16 +24,27 @@ By default, you can
 - connect to HTTP using the HTTP node
 
 By installing additional packages, you can:
-- connect to OPC-UA (node-red-controb-opcua)
-- ...
+- connect to OPC-UA (node-red-contrib-opcua)
+- connect to kafka (node-red-contrib-kafkajs)
+- connect to Siemens-S7 (node-red-contrib-s7)
+- connect to serial (node-red-node-serialport)
+
```

Comment:
There are many more: https://learn.umh.app/blog/node-red-in-industrial-iot-a-growing-standard/

https://flows.nodered.org/collection/U7bo_px1Nm6h
https://flows.nodered.org/node/node-red-contrib-cip-ethernet-ip

Also take a look in the industrial forum: https://discourse.nodered.org/c/industrial/18